ros2action/test/test_cli.py

```diff
@@ -135,7 +155,6 @@ def test_info_on_nonexistent_action(self):
             strict=False
         )
 
-    @launch_testing.markers.retry_on_failure(times=5)
```

Comment:
I believe this was a workaround for the race condition that I've fixed above and is no longer necessary